{"owner": {"display_name": "pythad", "reputation": 1581, "profile_image": "https://i.stack.imgur.com/k0Hxl.jpg?s=128&g=1", "user_id": 3832802, "user_type": "registered", "accept_rate": 90, "link": "http://stackoverflow.com/users/3832802/pythad"}, "link": "http://stackoverflow.com/questions/30743267/celery-module-object-has-no-attribute-app-when-using-python-3", "tags": ["python", "python-3.x", "celery"], "creation_date": 1433885788, "score": 1, "last_edit_date": 1434579011, "is_answered": true, "view_count": 729, "last_activity_date": 1477838403, "question_id": 30743267, "accepted_answer_id": 30903271, "answer_count": 2, "title": "Celery &#39;module&#39; object has no attribute &#39;app&#39; when using Python 3"}